src/main/java/mate/academy/hibernate/relations/service/impl/MovieServiceImpl.java

```diff
@@ -1,16 +1,26 @@
 package mate.academy.hibernate.relations.service.impl;
 
+import mate.academy.hibernate.relations.dao.impl.MovieDaoImpl;
+import mate.academy.hibernate.relations.exeptions.DataProcessingException;
 import mate.academy.hibernate.relations.model.Movie;
 import mate.academy.hibernate.relations.service.MovieService;
 
 public class MovieServiceImpl implements MovieService {
+    private final MovieDaoImpl movieDao;
```

Comment:
You should use the interface MovieDao instead of the implementation MovieDaoImpl to follow the Dependency Inversion principle. This will make the code more flexible and easier to maintain.